controllers/performanceprofile_controller.go

```diff
@@ -87,7 +87,8 @@ func (r *PerformanceProfileReconciler) SetupWithManager(mgr ctrl.Manager) error
 			kubeletOld := e.ObjectOld.(*mcov1.KubeletConfig)
 			kubeletNew := e.ObjectNew.(*mcov1.KubeletConfig)
 
-			return !reflect.DeepEqual(kubeletOld.Status.Conditions, kubeletNew.Status.Conditions)
+			return kubeletOld.GetGeneration() != kubeletNew.GetGeneration() ||
```

Comment:
how did this sneak in this PR?

Reply:
Just fixed on the way, because the logic is not correct, do you want a separate commit or PR for it?

Reply:
please move this (important) fix in its own commit. Please make sure to (briefly) explain the fix in the commit message.

Reply:
Moved to the separate PR - https://github.com/openshift-kni/performance-addon-operators/pull/806